katdal/ms_extra.py

```diff
@@ -372,6 +381,13 @@ def populate_main_dict(uvw_coordinates, vis_data, flag_data, weight_data, timest
     main_dict['FLAG_ROW'] = np.zeros(num_vis_samples, dtype=np.uint8)
     # The visibility weights
     main_dict['WEIGHT_SPECTRUM'] = weight_data
+    # Inverse visibility weights
```

Comment:
"Inverse square root visibility weights", just to be safe :-D

Or "Estimated RMS noise per frequency channel", like the SIGMA description.